README.md

```diff
@@ -1,111 +1,87 @@
-<p align="center">
-        <br />
-        <img src="https://www.inngest.com/assets/open-source/open-source-logo.svg" alt="Logo" width="220" height="90">
-</p>
-<p align="center">
-        The event-driven queue for any language.<br />
-        Created for rapid development speed and a delightful experience.
-</p>
+# Inngest
+
+![Latest release](https://img.shields.io/github/v/release/inngest/inngest?include_prereleases&sort=semver)
+![Test Status](https://img.shields.io/github/workflow/status/inngest/inngest/Go/main?label=tests)
+![Discord](https://img.shields.io/discord/842170679536517141?label=discord)
+![Twitter Follow](https://img.shields.io/twitter/follow/inngest?style=social)
+
+Inngest is an open-source, event-driven platform which makes it easy for developers to build, test, and deploy serverless functions without worrying about infrastructure, queues, or stateful services.
+
+Using Inngest, you can write and deploy serverless step functions which are triggered by events without writing any boilerplate code or infra. Learn more at https://www.inngest.com.
+
+- [Overview](#overview)
+- [Quick Start](#quick-start)
+- [Project Architecture](#project-architecture)
+- [Community](#community)
 
 <br />
 
 ## Overview
 
-Inngest makes it simple for you to write delayed or background jobs by triggering functions from events — decoupling your code from your queue.
+Inngest makes it simple for you to write delayed or background jobs by triggering functions from events — decoupling your code from your application.
 
-At a very high level, Inngest does two things:
+At the basic level, this is how you use Inngest:
```

Comment:
Might be able to remove this sentence entirely and go straight in to the bullets.